group_vars/matrix_servers

```diff
@@ -423,6 +423,45 @@ matrix_mautrix_telegram_database_password: "{{ matrix_synapse_macaroon_secret_ke
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-bridge-mautrix-twitter
+#
+######################################################################
+
+# We don't enable bridges by default.
+matrix_mautrix_twitter_enabled: false
+
+matrix_mautrix_twitter_container_image_self_build: "{{ matrix_architecture not in ['amd64', 'arm64'] }}"
+
+matrix_mautrix_twitter_systemd_required_services_list: |
+  {{
+    ['docker.service']
+    +
+    (['matrix-synapse.service'] if matrix_synapse_enabled else [])
+    +
+    (['matrix-postgres.service'] if matrix_postgres_enabled else [])
+    +
+    (['matrix-nginx-proxy.service'] if matrix_nginx_proxy_enabled else [])
+  }}
+
+matrix_mautrix_twitter_appservice_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'twt.as.token') | to_uuid }}"
+
+matrix_mautrix_twitter_homeserver_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'twt.hs.token') | to_uuid }}"
+
+matrix_mautrix_twitter_login_shared_secret: "{{ matrix_synapse_ext_password_provider_shared_secret_auth_shared_secret if matrix_synapse_ext_password_provider_shared_secret_auth_enabled else '' }}"
+
+# We'd like to force-set people with external Postgres to SQLite, so the bridge role can complain
+# and point them to a migration path.
+matrix_mautrix_twitter_database_engine: "{{ 'postgres' if matrix_postgres_enabled else 'sqlite' }}"
```

Comment:
Since we don't support `sqlite` for this bridge, maybe this should either pass an empty string and then have `validate_config.yaml` tell the user that Postgres is required and that they need to:
- set `matrix_mautrix_twitter_database_engine: postgres`
- and redefine the other `matrix_mautrix_twitter_database_*` variables

Reply:
Oops, I'd missed that sqlite reference. I'm down a different rabbit-hole right now, but will try to look at this tomorrow.

Reply:
So I had every intention to get back to this, but life happened, as it does, and it hasn't happened yet. I think I got a bit confused as to how best to make these changes you suggested, @spantaleev, and put it in the too hard basket until I had the energy to come back and get some more detailed. So I guess I'm just flagging that I haven't forgotten about this, but I also haven't done anything further on it because I felt a little stuck.

Also, I've just now rebased this PR on top of the latest `master` commit.